### PR TITLE
IPC Parts + Mechfab Designs Organization

### PR DESCRIPTION
**Pull request by DoctorSquishy**
# About The Pull Request
Adds all the IPC parts to be printable at Roboticist MechFabricator
Separates Mechfab designs into their own file category since that file was large
Construct an IPC from its core chest the same way you would take it apart, but in reverse! So, better be ready to learn how to do IPC surgery to insert the proper IPC organs.

## Why It's Good For The Game
Better clarity and organization of MechFabricator designs.
You can print an IPC out and put it together.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Printed out parts and put limbs together like normal, just need to insert organs and a "brain"
image

</details>

## Changelog

:cl: DoctorSquishy
add: Added IPC Part Printing expansion and IPCs can be built now from Roboticist MechFabricator
tweak: IPC Parts is now tech tier 4
refactor: Refactored MechFabricator for better organization in code
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -677,7 +677,20 @@
 	display_name = "IPC Parts"
```
Comment (nednaZ) [inline code/modules/research/techweb/all_nodes.dm:677]: Strangely this is a tier 3 tech, but it has a tier 4 requirement.

**Review by nednaZ — CHANGES_REQUESTED** [review]
Seems to work well, I just want the tech requirement made consistent.

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (DoctorSquishy) [reply code/modules/research/techweb/all_nodes.dm:677]: Changed to tier 4

**Review by nednaZ — APPROVED** [review]
All good.
Build yourself an IPC.

**Pull request merged**